srcalc/src/main/java/gov/va/med/srcalc/web/SrcalcUrls.java

```diff
@@ -48,4 +48,17 @@ public String getModelAdminHome()
     {
         return MODEL_ADMIN_HOME;
     }
+    
+    /**
+     * The URL of the Summary Report.
+     */
+    public static final String SUMMARY_REPORT = "/admin/reports/summary";
+    
+    /**
+     * Returns {@link #SUMMARY_REPORT}.
+     */
+    public String getSummaryReport()
```

Comment:
I'm not sure how you feel about the groupings in this class, but I usually put all constants at the top and then getters at the bottom. It doesn't need to change, but it's different than what typically is done.